app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt

```diff
@@ -0,0 +1,28 @@
+package com.prayercompanion.prayercompanionandroid.data.utils
```

Comment:
this class fits more in the domain layer package, can you move it to `prayercompanionandroid.domain.utils`

Reply:
resolved in [9d51ebe](https://github.com/Prayer-Companion/PrayerCompanionAndroid/pull/19/commits/9d51ebe66ae0172d935f9658bb85aa752c034953)